en/micro-integrator/docs/setup/observability/observability-deployment-strategy.md

```diff
@@ -6,8 +6,8 @@ WSO2 Enterprise Integrator 7.1 offers two observability solutions for monitoring
 
 The cloud-native solution is more suitable in the following scenarios:
 
-- If you are creating a new cloud-native Micro Integrator deployment. Refer the <a href="../../../setup/deployment/kubernetes_deployment_patterns">Micro Integrator Kubernetes Deployment</a> guide for details on how to setup a cloud-native Micro Integrator deployment on Kubernetes.
-- If you already have Prometheus, Grafana, and Jaeger as you in-house monitoring and observability tools.
+- If you are creating a new cloud-native Micro Integrator deployment. See the instructions on setting up a cloud-native [Micro Integrator deployment on Kubernetes](../../../setup/deployment/kubernetes_deployment_patterns).
+- If you already have Prometheus, Grafana, and Jaeger as your in-house monitoring and observability tools. This applies to both [VM deployments](../../../setup/deployment/deploying_wso2_ei) as well as [Kuberentes deployments](../../../setup/deployment/kubernetes_deployment_patterns).
```

Comment:
If you are creating a new cloud-native Micro Integrator deployment, see the instructions on setting up a cloud-native [Micro Integrator deployment on Kubernetes](../../../setup/deployment/kubernetes_deployment_patterns).